dockerfiles/base/scripts/base/startup_02_pre_docker.sh

```diff
@@ -428,7 +428,10 @@ init_check_mounts() {
   fi
 
   # Verify that we can write to the host file system from the container
-  check_host_volume_mount
+  if ! is_fast; then 
+    check_host_volume_mount
+  fi
+return 2
```

Comment:
Oops - indentation is correct.  That return statement should not be there.  Removed.